src/objects.c

```diff
@@ -518,6 +572,35 @@ static PgPool *new_pool(PgDatabase *db, PgUser *user)
 	return pool;
 }
 
+
+/* create new pool object */
```

Comment:
maybe add: for forwarding cancellations to other pgbouncers behind the same load balancer